src/applications/vaos/services/appointment/transformers.js

```diff
@@ -385,34 +370,47 @@ function setContained(appt) {
   switch (getAppointmentType(appt)) {
     case APPOINTMENT_TYPES.vaAppointment: {
       if (isVideoVisit(appt)) {
-        return [
-          {
-            resourceType: 'HealthcareService',
-            id: `HealthcareService/var${appt.vvsAppointments[0].id}`,
-            type: [
-              {
-                text: 'Patient Virtual Meeting Room',
-              },
-            ],
-            location: {
-              reference: `Location/var${appt.facilityId}`,
+        const contained = [];
+        const service = {
+          resourceType: 'HealthcareService',
+          id: `HealthcareService/var${appt.vvsAppointments[0].id}`,
+          type: [
+            {
+              text: 'Patient Virtual Meeting Room',
             },
-            characteristic: [
-              {
-                coding: getVideoType(appt),
-              },
-            ],
-            telecom: [
-              {
-                system: 'url',
-                value: getVideoVisitLink(appt),
-                period: {
-                  start: getMomentConfirmedDate(appt).format(),
+          ],
+          providedBy: {
+            reference: `Organization/var${appt.facilityId}`,
+          },
+          characteristic: [
+            {
+              coding: [
+                {
+                  system: 'VVS',
+                  code: appt.vvsAppointments[0].appointmentKind,
```

Comment:
I needed to pass through the appointment kind, so I tweaked this coding to create a "VVS" code system. Now we can detect video appointments by looking for a VVS characteristic (helper functions were updated to reflect this).